src/test/java/uk/nhs/hee/tis/trainee/forms/repository/S3FormRPartBRepositoryImplTest.java

```diff
@@ -250,6 +259,33 @@ void shouldGetFormRPartBsByTraineeTisId() {
         is(LifecycleState.UNSUBMITTED));
   }
 
+  @Test
+  void shouldParseSubmissionDateWhenInDateFormat() {
+    when(s3Mock.listObjects(bucketName, DEFAULT_TRAINEE_TIS_ID + "/forms/formr-b"))
+        .thenReturn(s3ListingMock);
+    S3ObjectSummary s3Summary = new S3ObjectSummary();
+    s3Summary.setKey(KEY);
+    String otherKey = KEY + "w/error";
+    S3ObjectSummary errorSummary = new S3ObjectSummary();
```

Comment:
It doesn't feel like the intention of this test is to verify the Amazon exception handling, `errorSummary` could be removed.